configupdater.go

```diff
@@ -68,5 +70,67 @@ func (pd *PopData) ConfigUpdater(conf *Config, stopch chan struct{}) {
 }
 
 func (pd *PopData) ProcessTapirGlobalConfig(gconfig tapir.GlobalConfig) {
-	log.Printf("TapirProcessGlobalConfig: %+v", gconfig)
+    log.Printf("TapirProcessGlobalConfig: %+v", gconfig)
+
+    if len(gconfig.Topics) != 1 {
+        POPExiter("ProcessTapirGlobalConfig: Error, multiple topics not supported")
+    }
+
+    // Assume there is only one topic and that it is the one we want
+    // TODO maybe sanitize or sanity check or something
+    newTopic := gconfig.Topics[0]
+    bootstrapServers := gconfig.Bootstrap.Servers
+    bootstrapUrl := gconfig.Bootstrap.BaseUrl
+    bootstrapKey := gconfig.Bootstrap.ApiKey
+
+    brokerName := viper.GetString("tapir.mqtt.observations-from-core.name")
```

Comment:
Jag tror att du behöver en uppdaterad sample config som beskriver vad detta är. Jag tror att du med brokerName menar namnet på den source som vi använder för observations (dvs normalt "dns-tapir"). Men det är ju inte en [MQTT] broker, det är bara ett namn...

Reply:
Updated. Iterates through all MQTT-sources and updates the topic of those who have not been explicitly marked as "immutable".